README.md

```diff
@@ -28,6 +28,7 @@ import { Notification } from "https://deno.land/x/notifier/whatwg/mod.ts";
 new Notification("Hello", {
   body: "World",
   icon: "/path/to/icon.png",
+  sound: "device-added" // only support Linux and macOS
```

Comment:
Could you remove this line? It would probably cause a type error

```suggestion
```


Instead, could you add the following code to the previous code block?

```typescript
await notify({
  title: "This is a title",
  message: "This is a message",
  sound: "device-added", // only supported in Linux and MacOS
});
```